support/build.gradle

```diff
@@ -4,12 +4,17 @@ buildscript {
         google()
         jcenter()
     }
-
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        // Requires Gradle 4.4+
```

Comment:
Why does the comment say `4.4+` and the classpath `3.1.3`?

Reply:
We have 2 version there. One is the version of Gradle, and the other one is the version of Gradle plugin **for Android**.
I designated the plugin version to `3.1.3` and Gradle `4.4+` can support the plugin. 

I thought commenting those detail might be confused and placed a URL, but seems like I was wrong. I will comment them either.